packages/circuit-ui/styles/style-mixins.ts

```diff
@@ -229,17 +210,9 @@ export function focusOutline(
  * the user agent determines via heuristics that the focus should be
  * made evident on the element.
  */
-// TODO: Simplify the function signature in the next major.
-export function focusVisible(
-  options: 'inset',
-): (args?: ThemeArgs) => SerializedStyles;
-export function focusVisible(args?: ThemeArgs): SerializedStyles;
-export function focusVisible(
-  argsOrOptions?: ThemeArgs | 'inset',
-): SerializedStyles | ((args?: ThemeArgs) => SerializedStyles) {
-  if (typeof argsOrOptions === 'string') {
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    return (_args?: ThemeArgs): SerializedStyles => css`
+export function focusVisible(options?: 'inset' | ThemeArgs): SerializedStyles {
```

Comment:
I kept the `ThemeArgs` type here despite it not being used so `focusVisible` can be passed directly to Emotion's `styled` function.